fili-core/src/test/groovy/com/yahoo/bard/webservice/data/cache/MemcachedCacheSpec.groovy

```diff
@@ -160,7 +160,7 @@ class MemcachedCacheSpec extends Specification {
             .request().get(String.class)
 
         then: "should read old result from cache"
-        GroovyTestUtils.compareJson(result, expected)
+        GroovyTestUtils.compareJson(result, '{"rows":[]}')
```

Comment:
"Deprecate CacheV1 components and remove from DruidWorkflow" from https://github.com/yahoo/fili/issues/255

<img width="1034" alt="screen shot 2017-05-31 at 11 57 20 am" src="https://cloud.githubusercontent.com/assets/16126939/26648745/575dc176-45f8-11e7-99e8-bc4b2d0211a0.png">